ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__gcp-4.15-nightly-x86.yaml

```diff
@@ -23,7 +23,7 @@ resources:
       memory: 200Mi
 tests:
 - as: control-plane-etcd-fips-24nodes
-  cron: 0 6 2 * *
+  cron: 0 6 9 * *
```

Comment:
Moving the day changes the balance.

I suggest: `0 10 9 * *`

Reply:
Done.